docs/html/user_guide.rst

```diff
@@ -1306,6 +1306,238 @@ issue tracker`_ if you believe that your problem has exposed a bug in pip.
+``pip install`` command. This new behaviour means that pip works harder
+to find out which version of a package is a good candidate to install.
+
+What does this behaviour look like?
+-----------------------------------
+
+Right now backtracking looks like this:
+
+::
+
+   $ pip install tea==1.9.8
+   Collecting tea==1.9.8
+   Downloading tea-1.9.8-py2.py3-none-any.whl (346 kB)
+   |████████████████████████████████| 346 kB 10.4 MB/s
+   Collecting spoon==2.27.0
+   Downloading spoon-2.27.0-py2.py3-none-any.whl (312 kB)
+   |████████████████████████████████| 312 kB 19.2 MB/s
+   Collecting hot-water>=0.1.9
+   Downloading hot-water-0.1.13-py3-none-any.whl (9.3 kB)
+   Collecting cup>=1.6.0
+   Downloading cup-3.22.0-py2.py3-none-any.whl (397 kB)
+   |████████████████████████████████| 397 kB 28.2 MB/s
+   INFO: pip is looking at multiple versions of this package to determine
+   which version is compatible with other requirements.
+   This could take a while.
+   Downloading cup-3.21.0-py2.py3-none-any.whl (395 kB)
+   |████████████████████████████████| 395 kB 27.0 MB/s
+   Downloading cup-3.20.0-py2.py3-none-any.whl (394 kB)
+   |████████████████████████████████| 394 kB 24.4 MB/s
+   Downloading cup-3.19.1-py2.py3-none-any.whl (394 kB)
+   |████████████████████████████████| 394 kB 21.3 MB/s
+   Downloading cup-3.19.0-py2.py3-none-any.whl (394 kB)
+   |████████████████████████████████| 394 kB 26.2 MB/s
+   Downloading cup-3.18.0-py2.py3-none-any.whl (393 kB)
+   |████████████████████████████████| 393 kB 22.1 MB/s
+   Downloading cup-3.17.0-py2.py3-none-any.whl (382 kB)
+   |████████████████████████████████| 382 kB 23.8 MB/s
+   Downloading cup-3.16.0-py2.py3-none-any.whl (376 kB)
+   |████████████████████████████████| 376 kB 27.5 MB/s
+   Downloading cup-3.15.1-py2.py3-none-any.whl (385 kB)
+   |████████████████████████████████| 385 kB 30.4 MB/s
+   INFO: pip is looking at multiple versions of this package to determine
+   which version is compatible with other requirements.
+   This could take a while.
+   Downloading cup-3.15.0-py2.py3-none-any.whl (378 kB)
+   |████████████████████████████████| 378 kB 21.4 MB/s
+   Downloading cup-3.14.0-py2.py3-none-any.whl (372 kB)
+   |████████████████████████████████| 372 kB 21.1 MB/s
+   Downloading cup-3.13.1-py2.py3-none-any.whl (381 kB)
+   |████████████████████████████████| 381 kB 21.8 MB/s
+   This is taking longer than usual. You might need to provide the
+   dependency resolver with stricter constraints to reduce runtime.
+   If you want to abort this run, you can press Ctrl + C to do so.
+   Downloading cup-3.13.0-py2.py3-none-any.whl (374 kB)
+
+In the above sample output, pip must downloaded multiple versions of
+package cup - cup-3.22.0 to cup-3.13.0 - to find a version that will be
+compatible with the other packages - ``spoon``, ``hot-water``, ``cup``
+etc.
+
+These multiple ``Downloading cup-version`` lines shows pip backtracking.
```

Comment:
```suggestion
These multiple ``Downloading cup-version`` lines show pip backtracking.
```

Reply:
This still needs to be fixed. :-)

Reply:
I had to reread this multiple times before I could actually see what was wrong here. 😫